extmap_test.go

```diff
@@ -0,0 +1,23 @@
+package sdp
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExtmap(t *testing.T) {
+	tests := []struct {
+		parameter string
+		expected  string
+	}{
+		{exampleAttrExtmap1, exampleAttrExtmap1Line},
```

Comment:
@trivigy did you forget to include these example strings?

Reply:
Nope. All there, they are just organized all in one file.

https://github.com/pions/sdp/pull/16/files#diff-0fe0a638de3620b2eaa11548e8c8ffcbR144

Reply:
cool, missed that. Thanks.